### DISCUSSION TITLE
Is there an way to create a map input for an selected image?

### DISCUSSION THREAD

**Discussion by scolastico** [directus/directus#12513] [Archive]
Hi, i want to create an input where you can select an position for an image which should also be selectable by an user. Is there a way to achieve this? I think its for some users to complicated to get the px coordinates of images.

**Comment by benhaynes** [comment]
You should be able to just add our Map Interface to Directus Files to get this. However, a hook would be required if you want to automatically pull location metadata from the file/image on upload.
